web-components/src/components/inputs/validation.ts

```diff
@@ -44,6 +45,10 @@ export function validatePassword(password: string): boolean {
   return hasUpperCase && hasLowerCase && hasNumber && hasSpecialCharacter;
 }
 
+const decimalCount = (num: number): number => {
+  return num.toString().split(/\./)?.[1]?.length;
```

Comment:
Should we extract the regex in a variable just like the other one at the top of the file?